extension/src/popup/views/ReviewAuth/index.tsx

```diff
@@ -332,15 +336,53 @@ const AuthDetail = ({
       // eslint-disable-next-line
       for (const transfer of transfers) {
         try {
-          // eslint-disable-next-line
-          const response = await fetch(tokenDetailsUrl(transfer.contractId));
+          if (isCustomNetwork(networkDetails)) {
```

Comment:
on standalone networks, use sorobanClient.server to fetch details rather than the BE